packages/psd/src/classes/Psd.ts

```diff
@@ -49,9 +50,14 @@ export class Psd extends Synthesizable implements NodeBase<never, NodeChild> {
         switch (resource.id) {
           case ResourceType.GridAndGuides:
             this.guides = resource.resource.guides;
-            break;
+            continue;
```

Comment:
Is it okay for change `break` statement into `continue` statement?

Reply:
Very good question. When I first looked at the code I assumed the original intention was to "map over all resources and assign them to proper slots". In this spirit `break` seems like a weird choice, since it'd stop the iteration early (before reading all resources).

But it's also possible that intention was different and/or I misunderstand how switch cases work in TS.

Tests pass in either case and there's not really much history in Git to see what the intention was. What do you think would be best?

Reply:
Ahh, I see - I misunderstood JS in this case. The `break` would refer to the switch statement, whilst `continue` applies to for-loop. Ugh. Reverting :)